instant-xml/src/lib.rs

```diff
@@ -185,11 +256,14 @@ impl<'xml, W: std::fmt::Write> Serializer<'xml, W> {
     }
 }
 
+#[derive(Clone)]
 pub enum FieldAttribute<'xml> {
     Prefix(&'xml str),
     Namespace(&'xml str),
+    Attribute,
 }
 
+#[derive(Clone)]
```

Comment:
Why do we need this?

Reply:
Here: https://github.com/InstantDomain/instant-xml/blob/022eed647090451e416ffec6b5b16880701759a5/instant-xml/src/lib.rs#L188

Reply:
Okay, you can use `take()` instead.